client/blocks/login/social.jsx

```diff
@@ -54,7 +54,13 @@ class SocialLoginForm extends Component {
 		const { onSuccess, socialService } = this.props;
 		let redirectTo = this.props.redirectTo;
 
-		if ( ! response.Zi || ! response.Zi.access_token || ! response.Zi.id_token ) {
+		if ( ! response.getAuthResponse ) {
```

Comment:
Nice fix!!!

Could we add it to https://github.com/Automattic/wp-calypso/blob/master/client/blocks/signup-form/social.jsx#L47 as well?

Reply:
Done, seems to work